transports/dns/src/lib.rs

```diff
@@ -21,17 +21,17 @@
 //! # [DNS name resolution](https://github.com/libp2p/specs/blob/master/addressing/README.md#ip-and-name-resolution)
 //! [`Transport`] for libp2p.
 //!
-//! This crate provides the type [`GenDnsConfig`] with its instantiations
-//! [`DnsConfig`] and `TokioDnsConfig` for use with `async-std` and `tokio`,
+//! This crate provides the type [`async_std::Config`] and [`tokio::Config`]
+//! for use with `async-std` and `tokio`,
 //! respectively.
 //!
-//! A [`GenDnsConfig`] is an address-rewriting [`Transport`] wrapper around
+//! A [`Config`] is an address-rewriting [`Transport`] wrapper around
 //! an inner `Transport`. The composed transport behaves like the inner
 //! transport, except that [`Transport::dial`] resolves `/dns/...`, `/dns4/...`,
 //! `/dns6/...` and `/dnsaddr/...` components of the given `Multiaddr` through
 //! a DNS, replacing them with the resolved protocols (typically TCP/IP).
 //!
-//! The `async-std` feature and hence the `DnsConfig` are
+//! The `async-std` feature and hence the `async_std::Config` are
```

Comment:
I think this section is actually heavily outdated. Doesn't need to be fixed now but would be appreciated :)